reana_job_controller/job_manager.py

```diff
@@ -130,3 +133,10 @@ def cache_job(self):
     def update_job_status(self):
         """Update job status in DB."""
         pass
+
+    def _extend_env_vars(self, env_vars):
+        """Extend environment variables with REANA specific ones."""
+        prefix = 'workflow'
+        env_vars[prefix + '_workspace'] = self.workflow_workspace
```

Comment:
Currently the PR creates variables in lowercase, e.g.:

```
workflow_workspace=/var/reana/users/00000000-0000-0000-0000-000000000000/workflows/aa6f5508-6145-40fd-ba08-c1b48b05c111
workflow_uuid=aa6f5508-6145-40fd-ba08-c1b48b05c111
```

This is unusual, we should (1) create them in uppercase and (2) perhaps let's prefix them with `REANA_...` so that they would be easily distinguishable from the user environment ones (that may contain thousands of other variables); see the discussion about using REANA-specific names in the issue. It'll be practical for users to be able to do `env | grep REANA_` in order to discover everything we set.